ecosystem/typescript/sdk/src/version.ts

```diff
@@ -0,0 +1,2 @@
+// hardcoded for now, we would want to have it injected dynamically
```

Comment:
Can't you do something like this?
```
import {version} from './package.json';
```

Reply:
not that easy, package.json is a configuration file and It doesn't contain any executable code that can be imported or used in the application, we need to do something like [this](https://github.com/alchemyplatform/alchemy-sdk-js/blob/main/inject-version.js) or [this](https://github.com/MystenLabs/sui/blob/main/sdk/typescript/genversion.mjs).
Also `import` used in esm modules while sdk supports both cjs and esm

Reply:
Okay unfortunate, let's roll with this for now.